lib/grpffmat.gi

```diff
@@ -125,11 +125,18 @@ end );
 ##
 #M  NiceMonomorphism( <ffe-mat-grp> )
 ##
+FULLGLNICOCACHE:=[];
```

Comment:
[Question] This will certainly require attention in HPC-GAP. Is this code covered by test? Codecov is not accessible right now, so I can't check.

Reply:
AH, yes. This is triggered as soon as a permutation representation for GL (or a subgroup on the whole space) is calculated, that will be part of existing tests.

Reply:
Thanks for comment - then in HPC-GAP that could be a thread local variable, so each thread can assign a distinct value to it: http://www.gap-system.org/Manuals/doc/hpc/chap2.html#X7D93681D7B5E8DCD